docker/ambulance-analytics-sandbox/src/routes/methodNotAllowed.js

```diff
@@ -0,0 +1,17 @@
+module.exports = [
+    /**
+     * GET,PUT,DELETE,PATCH to /submission
+     * If submissionResponse.json file exists in `mock` directory then that will be the response otherwise, 404
+     */
+    {
+      method: ['GET', 'PUT', 'DELETE', 'PATCH'],
```

Comment:
More of a question really - is it possible to do a negated search on the method name i.e. "all methods that are NOT post"?